src/Uno.UWP/Security/Authentication/Web/WebAuthenticationBrokerProvider.cs

```diff
@@ -21,7 +21,13 @@ public Uri GetCurrentApplicationCallbackUri()
 			{
 				// A default one is defined: we're using it instead of using
 				// a discovery mechanism.
+
+#if __ANDROID__
+				return defaultUri.TrimEndUriSlash();
```

Comment:
Still need to explain in the code why we need this on Android.

Reply:
It was removed.

Reply:
Please do not close the conversation until you've answered the comment or pushed a new branch that addresses the comment.

Reply:
The Android Platform does not handle the last slash in Uri automatically. It is native behavior. Did I answer your question @carldebilly ?

Reply:
@aesteves900 If you do have a link that explains this add it as a comment in the code, otherwise add `The Android Platform does not handle the last slash in Uri automatically.` If you have the error produced when adding the slash, can you add it in the comment as well? It can enhance the troubleshooting experience.